menuconfig.py

```diff
@@ -299,6 +299,141 @@
     "brightpurple": curses.COLOR_MAGENTA + 8,
 }
 
+def rgb_to_6cube(rgb):
+    """
+    Convert each component of a RGB color to its approximate index
+    in the xterm 6cube color palette
+    """
+    # The formula for converting a single RGB value to the 6cube index
+    rto6 = lambda x: 0 if x < 48 else max(1, int(round((x - 55) / 40)))
+
+    return [rto6(x) for x in rgb]
```

Comment:
With three parameters, this would become

    return (rto6(r), rto6(g), rto6(b))

Seems pretty nice and explicit, and you'd consistently deal with tuples.